crates/bevy_core_pipeline/src/fxaa/mod.rs

```diff
@@ -0,0 +1,256 @@
+mod node;
+
+use crate::{
+    core_2d, core_3d, fullscreen_vertex_shader::fullscreen_shader_vertex_state,
+    fxaa::node::FxaaNode,
+};
+use bevy_app::prelude::*;
+use bevy_asset::{load_internal_asset, HandleUntyped};
+use bevy_derive::Deref;
+use bevy_ecs::{prelude::*, query::QueryItem};
+use bevy_reflect::TypeUuid;
+use bevy_render::{
+    extract_component::{ExtractComponent, ExtractComponentPlugin},
+    prelude::Camera,
+    render_graph::RenderGraph,
+    render_resource::*,
+    renderer::RenderDevice,
+    texture::BevyDefault,
+    view::{ExtractedView, ViewTarget},
+    RenderApp, RenderStage,
+};
+
+#[derive(Eq, PartialEq, Hash, Clone, Copy)]
+pub enum Quality {
+    Low,
+    Medium,
+    High,
+    Ultra,
+}
+
+impl Quality {
+    pub fn get_str(&self) -> &str {
+        match self {
+            Quality::Low => "LOW",
+            Quality::Medium => "MEDIUM",
+            Quality::High => "HIGH",
+            Quality::Ultra => "ULTRA",
+        }
+    }
+}
+
+#[derive(Component, Clone)]
+pub struct Fxaa {
+    /// Enable render passes for FXAA.
+    pub enabled: bool,
+
+    /// The minimum amount of local contrast required to apply algorithm.
+    /// Use lower settings for a sharper, faster, result.
+    /// Use higher settings for a slower, smoother, result.
+    pub edge_threshold: Quality,
+
+    /// Trims the algorithm from processing darks.
+    /// Use lower settings for a sharper, faster, result.
+    /// Use higher settings for a slower, smoother, result.
+    pub edge_threshold_min: Quality,
+}
+
+impl Default for Fxaa {
+    fn default() -> Self {
+        Fxaa {
+            enabled: true,
+            edge_threshold: Quality::High,
+            edge_threshold_min: Quality::High,
+        }
+    }
+}
+
+impl Fxaa {
+    pub fn get_settings(&self) -> Vec<String> {
```

Comment:
What's the purpose of this? It's not used anywhere in this PR.

Reply:
It is no longer needed. Now removed.